docs/deprecated/pv-recycler.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docwithnav
+title: Persistent Storage Recycler Deprecation
+
+assignees:
+- thockin
+- childsb
+- erinboyd
+---
+
+## Feature description to be deprecated:
+
+When a user creates a persistent volume, they can specify the volume be 'recycled' and reused for a different container or user.
+The use case was driven by the fact 'users' did not have the permissions to create persistent volumes, only an admin can, and therefore
+recycling and reusing a volume again and again, seemed like a desirable feature.
+
+This option is specified in the persistent volume specification as follows:
+```yaml
```

Comment:
Add newlines before and after the yaml example to add some whitespace and make the formatting look correct.